src/nlmdb.h

```diff
@@ -41,6 +41,17 @@ static inline bool BooleanOptionValue(
     && optionsObj->Get(opt)->BooleanValue();
 }
 
+static inline bool BooleanOptionValueDef(
```

Comment:
I honestly don't know if it's necessary but I think we should put a `Nan::HandleScope scope;` in here and the three following functions. They are `inline` but I don't know if that provides any guarantees and adding an extra `HandleScope` isn't going to hurt even if it is unnecessary.